sass/directives/03_components/flyout/_flyout.scss

```diff
@@ -95,6 +95,7 @@
     &:hover {
       ul {
         display: block;
+        z-index: 1;
```

Comment:
Are we unable to use our z-index management here?

Reply:
Yes, it should look something like

`z-index: z($z-context, sticky-nav, $sticky-nav, sticky-nav);`

Reply:
@mrfantasticwonders is that function and variables available in stylebase?

Reply:
@MKwenhua This pattern appears to be used elsewhere in `style-base`, but it's unclear where the necessary variables come from. It may be placing a tight coupled dependency on Employer, which is bad: https://github.com/cb-talent-development/employer-style-base/blob/ce459d2547861dff05377bbc93aaffc2c24a26d9/sass/directives/03_components/flyout/_flyout.scss#L11

If you determine that these variables are not accessible directly in `style-base`, we can keep this as a hardcoded `z-index` value.